### ISSUE TITLE
Issues Reproducing Image PushT

### ISSUE THREAD

**Issue by thewindsofwinter** [jayLEE0301/vq_bet_official#11]
Recently was running some VQ-BeT baselines and was unable to get performance as specified in the paper for Image PushT (done by setting `visual_input: true` on the `nongoalcond` PushT). Attaching a screenshot of wandb plot (performance did not improve beyond `0.3` after this point) and expected performance in paper.

<img height="180" alt="image" src="https://github.com/user-attachments/assets/3a779dbc-f9f8-456a-900b-654927241b90" />
<img height="180" alt="image" src="https://github.com/user-attachments/assets/e036c3a2-7669-49fc-9b86-152c2c7e6a36" />

The following problem occurs when running `vq-bet` on `torch==2.1.0`, and is specific to Image PushT (for Image Kitchen and Lowdim PushT, results are close to reported values). I've attached the output of my code (with configuration) and package installs below for help in debugging: [debug.txt](https://github.com/user-attachments/files/18418237/debug.txt), [pip list.txt](https://github.com/user-attachments/files/18418238/pip.list.txt).

**Comment by thewindsofwinter** [comment]
Would appreciate help on any configuration differences or training recipe differences that lead to this discrepancy! For example, the `lerobot` repository integrates proprioception to achieve good image results, but was unsure whether this was in the original training recipe of the paper.

**Comment by jayLEE0301** [comment]
Hello @thewindsofwinter 
The discrepancy between the results come from this differences:
- the results of ImagePushT in the table are obtained with finetuned ResNet (i.e. we train ResNet with policy together)
- this git repo only provides 'pre-processed embedding vectors with pre-trained ResNet' using `resnet = models.resnet18(pretrained=True)`. (i.e. we don't update ResNet from pretrained model at all) -> this should take much less computational resources while has lower performance.

(All the baselines in the table for Image PushT are also evaluated with same configuration - train ResNet from scratch with the policy)

**Comment by thewindsofwinter** [comment]
Dear @jayLEE0301 

Thank you for your comment. Since the prepared data includes images as ResNet embeddings, it seems like gradients would not pass back through the ResNet during training even after `finetune_resnet=True`. Is there an original dataset that you used that we could use here (for reproducibility, in case there are slight differences from DP pushT dataset?)

**Comment by jayLEE0301** [comment]
Hello @thewindsofwinter 
we used the same dataset with DP pushT, but you may modify the code to train the ResNet from scratch, rather than reuse finetuned one.
